connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -281,7 +283,9 @@ void doShutdown() {
             Utils.closeQuietly(ctx, "connector context for " + connName);
             Utils.closeQuietly(metrics, "connector metrics for " + connName);
             Utils.closeQuietly(offsetStorageReader, "offset reader for " + connName);
-            Utils.closeQuietly(offsetStore::stop, "offset backing store for " + connName);
+            if (offsetStore != null) {
```

Comment:
Discovered this bug while working on https://github.com/apache/kafka/pull/12307 for https://issues.apache.org/jira/browse/KAFKA-14006.